ui/src/app/components/summary/summary.component.scss

```diff
@@ -24,7 +24,7 @@
   }
   .no-issue-container {
     width: 100%;
-    min-height: 400px;
+    min-height: 300px;
```

Comment:
Its not vertically aligned: https://drive.google.com/file/d/1VJ6FPpxoS5QRgjU4R0EWDJ4WJGrwtuIi/view?usp=share_link&resourcekey=0-bdDHfglZTtk-tp83j-yX6g

Reply:
Updated